episodes/introduction.md

```diff
@@ -1,114 +1,66 @@
 ---
-title: "Using Markdown"
+title: "Task Management for Humans"
 teaching: 10
 exercises: 2
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- How can you use tools and frameworks to work more effectively, not so you can do more work for your boss but as a form of self care?
```

Comment:
I suggest to replace boss -> employer

Reply:
Good point, will correct in the next check-in.